web/packages/teleport/src/Navigation/Navigation.tsx

```diff
@@ -128,10 +129,11 @@ function getDashboardNavigationSections(
   const navigationSections = features
     .filter(feature => feature.showInDashboard)
     .map(feature => ({
-      standalone: true,
-      title: feature.navigationItem.title,
-      Icon: feature.navigationItem.icon,
-      route: feature.navigationItem.getLink(cfg.proxyCluster),
+      standalone: {
+        title: feature.navigationItem.title,
+        Icon: feature.navigationItem.icon,
+        route: feature.navigationItem.getLink(cfg.proxyCluster),
+      },
```

Comment:
wouldnt these be `feature.navigationItem.standalone.title` etc?
also, what happens if route is defined in the `standalone` object, should we use that here instead? (or is it needed at all?)

Reply:
nevermind, i misread this. i see what is happening now. but i think route should not be undefined in the type then